src/api/cvc4cpp.h

```diff
@@ -3094,6 +3098,52 @@ class CVC4_PUBLIC Solver
    */
   std::vector<Term> getValue(const std::vector<Term>& terms) const;
 
+  /**
+   * Do quantifier elimination.
+   * SMT-LIB: ( get-qe <q> )
+   * Requires a logic that supports quantifier elimination. Currently, the only
+   * logics supported by quantifier elimination is LRA and LIA.
+   * @param q a quantified formula of the form:
+   *   Q x1...xn. P( x1...xn, y1...yn )
+   * where P( x1...xn, y1...yn ) is a quantifier-free formula
+   * @param strict whether to output warnings, such as when an unexpected logic
```

Comment:
Let's remove this argument.  Setting this to false is for internal use only, and always be fixed to `true` for users of the API.